### ISSUE TITLE
How to assure rsync support of "safe file lists" ?

### ISSUE THREAD

**Issue by usrflo** [perlundq/yajsync#20]
*yajsync currently supports a minimal subset of rsync protocol version 30.0, with the additional constraint that the peer must also support rsync safe file lists.*

How can I assure my native rsync client supports "safe file lists" ?

```
$ rsync --version
rsync  version 3.1.0  protocol version 31
Copyright (C) 1996-2013 by Andrew Tridgell, Wayne Davison, and others.
Web site: http://rsync.samba.org/
Capabilities:
    64-bit files, 64-bit inums, 64-bit timestamps, 64-bit long ints,
    socketpairs, hardlinks, symlinks, IPv6, batchfiles, inplace,
    append, ACLs, xattrs, iconv, symtimes, prealloc
```

**Comment by perlundq** [comment]
rsync version 3.1.0 is supposed to support safe file lists, but there was a bug early on in 3.1.0 which I reported to the rsync mailing list (with a fix):
  http://comments.gmane.org/gmane.network.rsync.general/25984

**Comment by perlundq** [comment]
This is fixed in commit b9e97b385643fe85704c412f5a0d09d5c0e35976
